File: 268-Missing-Number.js
function missingNumber(nums) {
    let output = 0
    nums.forEach((num, i) => {
        if(!nums.includes(i)) {            
            output = i
        }
        if(!nums.includes(nums.length)){
            output = nums.length
        }
    })
    return output
    
};

console.log(missingNumber([9,6,4,2,3,5,7,0,1]))

File: 268-Missing-Number.py
def MissingNumber(nums):
    output = 0
    for i, n in enumerate(nums):
        if not i in nums:
            output = i
        if not len(nums) in nums:
            output = len(nums)
    print(output)
    return output            

MissingNumber([1,2,5,4,0])